metpy/calc/thermo.py

```diff
@@ -1205,3 +1205,43 @@ def _isen_iter(iter_log_p, isentlevs_nd, ka, a, b, pok):
 
     # output values as a list
     return ret
+
+
+@exporter.export
+@check_units('[pressure]', '[temperature]', '[temperature]', '[temperature]')
+def sbcape_cin(pressure, temperature, dewpt, profile=None):
```

Comment:
Understood, but for a simple "starter" wrapper, I think keeping the parcel calculation hidden is best. Otherwise someone could pass a parcel that isn't surface based at all to the function and get a result that is indeed CAPE, but incorrect.